packages/@aws-cdk/aws-appsync/test/integ.graphql.expected.json

```diff
@@ -58,7 +58,7 @@
         "UserPoolName": "myPool",
         "VerificationMessageTemplate": {
           "DefaultEmailOption": "CONFIRM_WITH_CODE",
-          "EmailMessage": "Hello {username}, Your verification code is {####}",
+          "EmailMessage": "The verification code to your new account is {####}",
```

Comment:
I believe this message is also used for MFA verification, so perhaps drop the word 'new'. What do you think?

Reply:
I think your suggestion is great.

Reply:
@nija-at I think your suggestion is great, but existing code about `EmailSubject` inculde 'new'. Should I drop the word 'new' from `EmailSubject` too?

https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-cognito/lib/user-pool.ts#L698

Reply:
Actually, I am wrong here. Email is not used as MFA; I confused this one with SMS. I'm ok leaving this as it is.